docs/client-options.md

```diff
@@ -118,6 +118,7 @@ third party clients.
 - [Certify](https://github.com/webprofusion/Certify) GUI (.NET, WinForms)
 - [oocx/acme.net](https://github.com/oocx/acme.net) (.NET)
 - [kelunik/acme-client](https://github.com/kelunik/acme-client) (PHP)
+- ZeroSSL: [Portable Perl version](https://zerossl.com/installation.html#portablePerl), [Portable Win32/Win64 binaries](https://github.com/do-know/Crypt-LE/releases).
```

Comment:
I am indeed a developer behind ZeroSSL (ZeroSSL.com is also referring Do-Know.com quite a few times), so those are official binaries with MD5 and SHA256 checksums provided as well as part of the release information.

As for different "incarnations" of ZeroSSL - my GitHub profile says "passionate software developer" for a reason :) Once I have created a Perl client and later an online one, implementing Docker images and Win binaries seemed like a logical next step. Though probably I should have been doing something different today (on my holidays) :)

Re portable - what I meant is that those binaries are self-sufficient in terms of not depending on specific libraries in the system and not requiring installations. For example - some client might require a specific .Net version, having OpenSSL or PHP installed, which might make taking the client on a flash drive with you and running it some place else a bit problematic.

Anyways, I see your point and I tend to agree that it's best we keep it nice and tidy, so maybe 

- [ZeroSSL Windows](https://github.com/do-know/Crypt-LE/releases)

would work best and I can refer to Portable Perl installation instruction from the release page for those who might be interested in taking that route (for example if willing to create custom plugins).